kafka-0.10.x/src/main/scala/monix/kafka/KafkaConsumerConfig.scala

```diff
@@ -237,9 +244,10 @@ final case class KafkaConsumerConfig(
   retryBackoffTime: FiniteDuration,
   observableCommitType: ObservableCommitType,
   observableCommitOrder: ObservableCommitOrder,
-  observableSeekToEndOnStart: Boolean) {
+  observableSeekToEndOnStart: Boolean,
+  properties: Map[String, String]) {
 
-  def toMap: Map[String,String] = Map(
+  private val predefinedProperties = Map(
```

Comment:
I think it should stay `def`  because we don't have to create this `Map` if we never call `toMap` (e.g. we create Config and just update it few times)

Reply:
The validation happens every time case class is created, that's why I made it a val, can change if we don't want to run validations